scripts/install-cni.sh

```diff
@@ -131,19 +131,119 @@ else
 fi
 
 # Fill CNI spec template.
-ipv4_subnet=$(jq '.spec.podCIDR' <<<"$response")
+#######################################
+# Checks if given subnet is valid IPv4 range.
+# Arguments:
+#   Subnet
+# Returns:
+#   0 if valid, 1 on invalid.
+#######################################
+function is_ipv4_range {
+  local IPV4_VALIDATION_REGEXP='^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]{1,2}$'
+  local ip=$1
+
+  if [[ "${ip:-}" =~ ${IPV4_VALIDATION_REGEXP} ]] ; then
+    return 0
```

Comment:
This actually.. doesn't need an `if` and `return`. Just put `[[ "${ip:-}" =~ ${IPV4_VALIDATION_REGEXP} ]]` as the last statement and its return value is returned automatically.

Reply:
done